e2e-tests/gef/cypress/e2e/scenarios/maker-checker-user-role/maker_checker-should-not-submit-edited-deal-add-document.spec.js

```diff
@@ -37,12 +37,7 @@ context('Create application as MAKER, edit as MAKER_CHECKER, submit application
 
       // Make the deal an Automatic Inclusion Application
       applicationDetails.automaticCoverDetailsLink().click();
-      automaticCover.automaticCoverTerm().each(($el, index) => {
-        $el.find('[data-cy="automatic-cover-true"]').trigger('click');
-        if (index === 7) {
-          $el.find('[data-cy="automatic-cover-false"]').trigger('click');
-        }
-      });
+      cy.automaticEligibilityCriteria();
```

Comment:
It looks like one of the ECs was set to false before, is this a deliberate change?